services/ml/compilation_delegate_bnns.mm

```diff
@@ -525,6 +527,39 @@ void SetExtendInputs(const uint32_t extend_input_idx,
   return true;
 }
 
+bool CompilationDelegateBnns::CompileLogistic(
+    const mojom::ModelInfoPtr& model,
+    const mojom::OperationPtr& operation,
+    OperationMac& operation_bnns) {
+  DLOG(INFO) << "CompilationDelegateBnns::CompilLogistic";
+
+  operation_bnns.offset_x = 0;
+  operation_bnns.offset_y = 0;
```

Comment:
How about moving l.536 l.537 l.545 to a common section before calling compilation each operand, do specially if there are need, even `in_desc, out_desc`.